ocs_ci/ocs/ui/deployment_ui.py

```diff
@@ -172,6 +172,7 @@ def install_ocs_operator(self):
         logger.info("Click Install OCS")
         self.do_click(self.dep_loc["click_install_ocs"])
         self.do_click(self.dep_loc["click_install_ocs_page"])
+        time.sleep(60)
```

Comment:
Wouldn't be possible to do some slightly smarter check here, instead of waiting for hardcoded one minute?